docs/api/schema.md

```diff
@@ -6,15 +6,7 @@ A schema is what defines a Model and the validation performed on the collection
 
 ## `schema`
 
-Creates a schema for a model and define validation options for it.
-
-Read more about the validation [options](https://docs.mongodb.com/manual/core/schema-validation/index.html#behavior)
-in the MongoDB docs.
-
-Under the hood it is just a wrapper around the [`object`](api/types.md#object) type with some default properties
-(e.g. `_id` and timestamps properties).
-
-While the default `_id` property is added with an `ObjectId` type, its type can be customized into a `string` or a `number`
+Creates a schema for a model and define validation options for it.

Read more about the validation [options](https://docs.mongodb.com/manual/core/schema-validation/index.html#behavior)
in the MongoDB docs.

Under the hood it is just a wrapper around the [`object`](api/types.md#object) type with some default properties
(e.g. `_id` and timestamps properties).

While the default `_id` property is added with an `ObjectId` type, its type can be customized into a `string` or a `number`
```

Comment:
Do you mind fixing these newlines?

Reply:
Done. Not sure why, but I guess IntelliJ wants to do it's own thing with line endings lol VS Code handled it better it seems.